packages/@okta/vuepress-site/docs/reference/api/schemas/index.md

```diff
@@ -1533,6 +1539,155 @@ The following response is only a subset of properties for brevity.
 }
 ```
 
+## Log Stream Schema operations
+
+<ApiLifecycle access="ea" />
+
+> **Note:** The **Log Streaming** Early Access feature must be enabled. See [Manage Early Access and Beta features](https://help.okta.com/okta_help.htm?id=ext_Manage_Early_Access_features) for more information on Okta's Self-Service Feature Manager.
```

Comment:
```suggestion
> **Note:** The **Log Streaming** Early Access feature must be enabled. See [Manage Early Access and Beta features](https://help.okta.com/okta_help.htm?id=ext_Manage_Early_Access_features) for more information on Feature Manager.
```
The Product team is telling me to use "Feature Manager" and not Okta Self-Service Feature Manager